receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -226,19 +226,26 @@ func (jr *jReceiver) stopTraceReceptionLocked(ctx context.Context) error {
 }
 
 func (jr *jReceiver) SubmitBatches(ctx thrift.Context, batches []*jaeger.Batch) ([]*jaeger.BatchSubmitResponse, error) {
-	jbsr := make([]*jaeger.BatchSubmitResponse, 0, len(batches))
-	spansMetricsFn := internal.NewReceivedSpansRecorderStreaming(ctx, "jaeger-collector")
+	var err error
+	var nSpans int64
+
+	observabilityRecorder := internal.NewExporterEventRecorder("jaeger-collector")
+
+	octx := observabilityRecorder.Start(ctx, nil)
+	defer observabilityRecorder.End(octx, nSpans, err)
```

Comment:
same problem with `err` and `nSpans` -- I think this exists everywhere its used, so not going to comment on each place